12.0 [MIG] l10n_es_dua

acysos opened the pull request:
Hola,

Migración de l10n_es_dua a 12.0

Saludos

antespi committed:
[ADD] l10n_es_dua addon

oca-transbot committed:
OCA Transbot updated translations from Transifex

Commit:
[MIG] l10n_es_dua: Migrationt to 10.0 (#633)

Commit:
[MIG] l10n_es_dua: Migration to 11.0

[110][l10n_es_dua] Update contributors with up-to-date data

The same update as in v10 https://github.com/OCA/l10n-spain/pull/743

oca-travis committed:
[UPD] Update l10n_es_dua.pot

pedrobaeza commented:
Buenas, Ignacio, por favor permite que pueda empujar cambios a la rama para arreglar el historial de commits quitando los superfluos como los de renombrar maniesto, eliminar el en.po, etc.

pedrobaeza commented:
@acysos veo que has hecho tú el cambio, pero:

1. No has avisado del cambio y entonces me he enterado de chiripa por entrar de nuevo al PR. Avisa por favor la próxima vez.
2. Elimina también el commit "[MIG] Make modules uninstallable", que fue un commit de apoyo al anterior proceso y que ya ahora no aporta nada. Puede que te dé conflicto, pero es fácil de arreglar.
3. Fusiona "[110][l10n_es_dua] Update contributors with up-to-date data" con el de migración a v11, ya que de hecho fue una errata directa que se corrigió por interfaz.

Todo esto ya lo tenía yo hecho en local, pero al no dejarme hacer push, es doble trabajo...

pedrobaeza committed:
[FIX] l10n_es_dua: Flip name/description in DUA tax

They were flipped.

pedrobaeza commented:
Ignacio, vas a hacer los cambios pedidos? He descubierto en v11 este error:

https://github.com/OCA/l10n-spain/commit/a563f9a55bf70c8d5960ef61cd521538c66e1ce1

Que también deberías poner aquí, aunque estamos trabajando para quitarnos la descripción con código en los l10n_es_aeat*.

acysos commented:
Realizado

pedrobaeza commented on l10n_es_dua/__manifest__.py:
```diff
@@ -0,0 +1,31 @@
+              'Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/l10n-spain',
+    'license': 'AGPL-3',
+<<<<<<< HEAD
```
@acysos no resolviste bien el conflicto aquí.

acysos committed:
[MIG] l10n_es_dua: Migration to 12.0

[ADD] l10n_es_dua: restore en.po remove to drop commits

[ADD] l10n_es_dua: remove en.po

[ADD] l10n_es_dua: fragmented readme

pedrobaeza commented:
Travis dio un error, pero tuvo que ser un glitch temporal y ya está resuelto relanzando el trabajo.

Viendo el commit de migración (802ba77e7d5e9ed7d9404f965caa8b7e2d9b8b12), veo cosas raras como la resolución de conflictos ahí mismo y la eliminación del en.po. ¿Eso es que no limpiaste bien el historial de commits? También sería conveniente utilizar el README por fragmentos ya de cara a tener todos los módulos homogéneos en v12. Vuelvo a reiterar que si me das acceso a la rama, puedo hacerlo yo para quitarte el trabajo.

pedrobaeza commented:
@acysos si no avisas que has hecho forced push, nadie se entera de los cambios. De todas formas, sigo viendo varios problemas, en concreto, el comentario que hice de:

> veo cosas raras como la resolución de conflictos ahí mismo y la eliminación del en.po. ¿Eso es que no limpiaste bien el historial de commits?

La resolución de conflictos debía hacerse en el propio commit donde ocurre el conflicto, no luego al final, y lo de la eliminación del en.po igualmente es algún tipo de squash incorrecto con el último commit en lugar de con los "OCA Transbot commits".

Tardo commented:
Buenas, ¿este PR está cerrado definitivamente? ¿Se espera abrir otro?
Habría que actualizar: https://github.com/OCA/l10n-spain/issues/922

acysos commented:
Hay otro PR con este tema: https://github.com/OCA/l10n-spain/pull/1078

Tardo commented:
@acysos Gracias!
